src/OpenTK/Platform/X11/X11GLNative.cs

```diff
@@ -792,13 +861,15 @@ public override void ProcessEvents()
             // Process all pending events
             while (Exists && window != null)
             {
+                //Functions.XNextEvent(window.Display, ref e);
```

Comment:
When I wrote code lines below blocks X11 drop events. I added this line to test that and forgot to delete.